lib/logstash/outputs/amazon_es/elasticsearch-template-es7x.json

```diff
@@ -6,39 +6,37 @@
     "number_of_shards": 1
   },
   "mappings" : {
```

Comment:
can we also bump the version to say `60002`

Reply:
Updated version to 60002